### PR TITLE
Added cli flag to change overlay_cursor option, defaulting to false

### PR DESCRIPTION
**Pull request by danshick**
Adds the -p/--pointer flag, which allows the user to set the overlay_cursor property in the backend.

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -575,6 +575,7 @@ int wayvnc_usage(FILE* stream, int rc)
 "    -o,--output=<name>                        Select output to capture.\n"
 "    -k,--keyboard=<layout>                    Select keyboard layout.\n"
 "    -s,--seat=<name>                          Select seat by name.\n"
+"    -p,--pointer=true|false                   Select whether or not to overlay cursor (default to false).\n"
```
Comment (any1) [inline src/main.c]: Maybe call this render-cursor instead? It shouldn't take an argument.

File: src/main.c [hunk 2]
```diff
@@ -810,6 +817,10 @@ int main(int argc, char* argv[])
 		break;
 	}
 
+  self.capture_backend->overlay_cursor = false;
+  if(overlay_cursor && strncmp(overlay_cursor, "true", 4))
```
Comment (any1) [inline src/main.c]: I think you mean to say `strcmp(overlay_cursor, "true") == 0`, but that's not needed either.

File: src/main.c [hunk 3]
```diff
@@ -810,6 +817,10 @@ int main(int argc, char* argv[])
 		break;
 	}
 
+  self.capture_backend->overlay_cursor = false;
```
Comment (any1) [inline src/main.c]: The self structure is initialised to 0 at the top of this function, so this shouldn't be needed. Something strange is going on here.

File: src/main.c (hunk 2 above)
Comment (danshick) [reply src/main.c]: Yup, that's obsolete now, but yes I did.

**Comment by danshick** [comment]
How about now?

File: src/main.c (hunk 2 above)
Comment (danshick) [reply src/main.c]: I've been conditioned to use strncmp for improperly terminated inputs, but I meant to ! the second term.

File: src/main.c (hunk 3 above)
Comment (danshick) [reply src/main.c]: Yeah, it was initialized to false, my poorly written conditional above was faking me out.

**Comment by any1** [comment]
Much better. Can you squash the changes into one commit and force push? Please use imperative wording in the commit message. Something like "Add option to enable cursor overlay rendering".

**Comment by danshick** [comment]
Done, thanks

**Comment by any1** [comment]
Thanks!

**Pull request merged**